content/about/brand.md

```diff
@@ -140,6 +140,21 @@ grey-color-light-bg mb-2"></div>
 </div>
 
 
+### GRASS flyers
+
+<div>
+<p> For a simple overview of <b>GRASS GIS</b> you can download one of the two <b>GRASS GIS flyer</b> </p>
+<ul>
+  <li>
+      <b>OSGeo</b> one sheet flyer <a href="https://github.com/OSGeo/osgeo/raw/master/marketing/print/one-page-info/eu/grass_gis_a4.pdf" class="inl btn btn-primary" target="_blank">Download PDF</a>
+  </li>
+  <li><b>GRASS community</b> threefold flyer <a href="https://trac.osgeo.org/grass/export/74509/grass-promo/grassflyer/flyer_2016/en/grass_flyer.pdf" class="inl btn btn-primary" target="_blank">Download PDF</a>
```

Comment:
```suggestion
  <li><b>GRASS community</b> threefold flyer <a href="https://trac.osgeo.org/grass/export/74509/grass-promo/grassflyer/flyer_2016/en/grass_flyer.pdf" class="inl btn btn-primary" target="_blank">Download PDF</a>
```

better use a link to https://github.com/OSGeo/grass-promo/tree/master/grassflyer ?